src/pages/api/account.ts

```diff
@@ -4,25 +4,35 @@ import { Handler } from "../../@types/ApiPageHandler";
 import { addIdentityToUser } from "../../command/addIdentityToUser";
 import { removeIdentityFromUser } from "../../command/removeIdentityFromUser";
 import { withAPIPageLogger } from "../../middleware/withAPIPageLogger";
+import Sentry, { configureReq } from "../../utils/sentry";
 
 const handler: Handler = async (request, response) => {
-  const { userId, type, value } = request.body;
+  configureReq(request);
```

Comment:
Shouldn't we make a middleware?

Reply:
Probably, but I'd say we should work on the middleware composition before adding more middleware, don't you think?

The redirect middleware is also right around the corner.

I don't mind coding this middleware in this PR though.

Reply:
well, let's make that the next PR then

Reply:
Should I still refacto this into a middleware?

Reply:
Telephone log: we'll wait to have the middleware composition before refactoring this into a middleware.